backend/Application/Api/Rest/ExportController.cs

```diff
@@ -13,6 +13,7 @@ namespace Application.Api.Rest;
 public class ExportController : ControllerBase
 {
     private readonly IDbContextFactory<GraphQlDbContext> _dbContextFactory;
+    private const int queryLimit = 25000;
```

Comment:
May also be better that given we reach the limit to simply provide a 500 or something equivalent if it is better not obtaining any data rather than a subset of the data

Reply:
After talk with Simon we agreed that this should be a 500